.github/workflows/sync-public-with-private.yml

```diff
@@ -0,0 +1,27 @@
+name: Sync IC public with ic-private
+
+on:
+  schedule:
+    - cron: '*/10 * * * *'
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-latest
    if: ${{ github.repository != 'dfinity/ic' }}
```

Reply:
Well I thought it was easier to just disable the workflow on the public ic. That way it won't be triggered at all instead of having skipped runs every 10 minutes.

Reply:
Yes, that's easier. This config is just additional safeguard that makes sure that nothing ever runs on `dfinity/ic`.